app/assets/javascripts/discourse/lib/utilities.js.es6

```diff
@@ -660,5 +660,32 @@ export function postRNWebviewMessage(prop, value) {
   }
 }
 
+function reportToLogster(name, error) {
+  const data = {
+    message: `${name} theme/component is throwing errors`,
+    stacktrace: error.stack
+  };
+
+  Ember.$.ajax("/logs" + "/report_js_error", {
```

Comment:
Why not using our ajax lib?

Also why not ""/logs/report_js_error"" instead of `"/logs" + "/report_js_error"`

Reply:
It seems MiniRacer doesn't like importing `ajax` from the utilities file.

```
MiniRacer::RuntimeError: Error: Could not find module `discourse/lib/ajax` imported from `discourse/lib/utilities`
```

https://travis-ci.org/discourse/discourse/jobs/536272772#L7870

Works perfectly fine locally though.